polymorphia-frontend/components/xp-card/inner-components/types.ts

```diff
@@ -1,6 +1,7 @@
 export interface XPCardPointsProps {
   points: string;
   isSumVisible?: boolean;
+  isXPVisible?: boolean;
```

Comment:
```suggestion
  isSumLabelVisible?: boolean;
  isXPLabelVisible?: boolean;
```

I'd consider renaming these props to be more descriptive but it's a nitpick and I'm leaving this up to you.